src/main/java/org/fluentd/logger/sender/AFUNIXSocketSender.java

```diff
@@ -0,0 +1,230 @@
+package org.fluentd.logger.sender;
+
+import org.fluentd.logger.errorhandler.ErrorHandler;
+import org.msgpack.MessagePack;
+import org.newsclub.net.unix.AFUNIXSocket;
+import org.newsclub.net.unix.AFUNIXSocketAddress;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.BufferedOutputStream;
+import java.io.File;
+import java.io.IOException;
+import java.nio.ByteBuffer;
+import java.util.Map;
+
+@SuppressWarnings("Duplicates")
+public class AFUNIXSocketSender implements Sender {
```

Comment:
As you may know, this class has a lot of duplicated codes with RawSocketSender. So we'd better refactor them in the future...